packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -51,8 +51,10 @@ export interface OptimizeCiemssOperationState extends BaseState {
 	// Charts + Outputs:
 	chartConfigs: string[][];
 	inProgressOptimizeId: string;
-	inProgressForecastId: string;
-	forecastRunId: string;
+	inProgressPreForecastId: string;
+	preForecastRunId: string;
+	inProgressPostForecastId: string;
+	postForecastRunId: string;
```

Comment:
Your choice of names may be a bit confusing? Isn't it more like [pre-optimize, post-optimize], instead of [pre-forecast, post-forecast]?

Reply:
hmm yes but they are both IDs for forecasts.
[pre-optimized-forecast] is a bit wordy.
Maybe firstForecast, and secondForecast? Im not certain but am open to suggestions!

Reply:
Oh I think I am reading them as:

```
preForecast-runId
postForecast-runId
```
but you are implying
```
pre-forecastRunId
post-forecastRunId
```

Reply:
Correct